tensorcircuit/translation.py

```diff
@@ -81,6 +82,92 @@ def _merge_extra_qir(
             nqir += inds[k]
     return nqir
 
+class CustomizedCirqGate(cirq.Gate):
```

Comment:
one concern of this class outside translation function is that: if cirq is not installed, then the import already raise error since ``cirq.Gate`` is in the global scope when fails tc import

Reply:
So I suggest to move the class within the ``qir2cirq`` function

Reply:
Make sense